ui/app/main-template.html

```diff
@@ -0,0 +1,101 @@
+<div class="container" ng-cloak>
+  <div ng-show="isReady">
+    <div class="page-header">
+      <h1>SKY API Authorization Code Flow Tutorial</h1>
+    </div>
+
+    <!--
+      LOGIN
+        This section is hidden if the session has NOT been authenticated.
+    -->
+    <div ng-if="!isAuthenticated">
+      <div class="well">
+        <h4 class="well-title">Applications implementing the authorization code flow may be authorized in one of two ways:</h4>
```

Comment:
Language adjusted here to denote the following as "options", and not the only possible ways to authorize a SKY API application.